pywbemtools/pywbemcli/_cmd_statistics.py

```diff
@@ -0,0 +1,347 @@
+    'statistics' keyword.
+    """
+    pass  # pylint: disable=unnecessary-pass
+
+
+@statistics_group.command('reset', cls=PywbemcliCommand,
+                          options_metavar=CMD_OPTS_TXT)
+@add_options(help_option)
+@click.pass_obj
+def statistics_reset(context):
+    """
+    Reset the values in the client local statistics.
+
+    This command does not reset statistics in the WBEM server but
+    it does reset the statistics kept in pywbemcli on the
+    ``WBEMServerResponseTime``.
+    """
+    # pylint: disable=too-many-function-args
+    context.execute_cmd(lambda: cmd_statistics_reset(context))
+
+
+@statistics_group.command('server-on', cls=PywbemcliCommand,
+                          options_metavar=CMD_OPTS_TXT)
+@add_options(help_option)
+@click.pass_obj
+def statistics_server_on(context):
+    """
+    Enable statistics on current server.
+
+    This may fail if the server does not manage statistics or does not allow
+    a client to modify the state of statistics gathering.
+
+    This does not impact pywbem local client statistics gathering. See the
+    --timestats general option.
+    """
+    # pylint: disable=too-many-function-args
+    context.execute_cmd(lambda: cmd_statistics_server_on(context))
+
+
+@statistics_group.command('server-off', cls=PywbemcliCommand,
+                          options_metavar=CMD_OPTS_TXT)
+@add_options(help_option)
+@click.pass_obj
+def statistics_server_off(context):
+    """
+    Disable statistics on current server.
+
+    This may fail if the server does not manage statistics or does not allow
+    a client to modify the state of statistics gathering.
+
+    This does not impact pywbem local client statistics gathering. See the
+    --timestats general option.
+    """
+    # pylint: disable=too-many-function-args
+
+    context.execute_cmd(lambda: cmd_statistics_server_off(context))
+
+
+@statistics_group.command('server-show', cls=PywbemcliCommand,
+                          options_metavar=CMD_OPTS_TXT)
+@add_options(help_option)
+@click.pass_obj
+def statistics_server_show(context):
+    """
+    Display statistics gathered by server.
+
+    Display the current statistics gathered by the server if statistics
+    gathering is implemented for the current WBEM server.
+
+    Presents a table of the gathered statistical information.
+
+    This is not implemented. See issue #895
+    """
+    # pylint: disable=too-many-function-args
+    context.execute_cmd(lambda: cmd_statistics_server_show(context))
+
+
+@statistics_group.command('show', cls=PywbemcliCommand,
+                          options_metavar=CMD_OPTS_TXT)
+@add_options(help_option)
+@click.pass_obj
+def statistics_show(context):
+    """
+    Display statistics gathered by client.
+
+    Display the current statistics including local client gathered statistics
+    on WBEM operations and if server statistics are enabled, the
+    WBEMServerResponseTime statistics.
```

Comment:
Let's consider the server response time to be part of the client statistics although its return must be enabled on the server. Also, let's use the term "client statistics". How about replacing this sentence with:

"Display the client statistics gathered by pywbemcli. If the returning of server response time is implemented and enabled on the server, the server response time will be included in the client statistics."

Reply:
Done.  This was another one I rewote